README.md

```diff
@@ -23,38 +24,35 @@ metadata:
   name: my-secret-with-ovirt-credentials
 type: Opaque
 stringData:
-  config.yaml: |-
+  ovirt: |-
     apiUrl: "https://ovirt-engine.example.com:8443/ovirt-engine/api/"
     username: admin@internal # provided in the format of username@domain
     password: 123456
     insecure: true
```

Comment:
I think that setting this flag is not needed. If the `ca` is not provided we should set insecure to `true`.

Reply:
@machacekondra mentioned that without specifying  `insecure`, the ovirt-client will attempt using system CA. For insecure, it is necessary to provide it.

@machacekondra pls correct me if i'm wrong

Reply:
We need to set it. My point is why user needs to provide. We can figure it out based on ca availability and hide this detail from a user.

Reply:
Well the logic is:
1) if user provide CA use provided CA
2) if user don't provide CA use system store
3) if user use insecure = true, use insecure connection.

Meaning, that user must explicitly say to use insecure connection, and don't use it by default.

Reply:
I think in a container we won't be able to use second point.